### PR TITLE
Add missing container to whaletainership. fixes #821

### LINKED ISSUE #821: Inaccurate logo in ASCII representation
The ASCII representation of the Docker whaletainership is missing a container:

ascii

vs

original

Proposed solution: Add `##` to whale.

### PR DESCRIPTION
**Pull request by abrkn**
Closes #821

### CODE REVIEW

**Comment by thaJeztah** [comment]
Can you edit the description on this page so that it includes the text "closes #821"? Otherwise GitHub won't automatically close the issue if this is merged. (GitHub doesn't look at the *title* of the pull-request)

**Comment by abrkn** [comment]
:+1:

File: rootfs/rootfs/usr/local/etc/motd [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-                  ## ## ##       ==
-               ## ## ## ##      ===
-           /""""""""""""""""\___/ ===
-      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
-           \______ o          __/
-             \    \        __/
-              \____\______/
+                        ##         .
+                  ## ## ##        ==
+               ## ## ## ##       ===
```
Comment (tianon) [inline rootfs/rootfs/usr/local/etc/motd]: This one seems to be missing the new container. :smile:

**Comment by abrkn** [comment]
I actually managed to extend the whale without adding the container. Thanks for the heads up, @tianon

**Comment by tianon** [comment]
LGTM

**Pull request merged**